build-tools/packages/build-cli/docs/generate.md

```diff
@@ -355,6 +356,28 @@ EXAMPLES
 
 _See code: [src/commands/generate/releaseNotes.ts](https://github.com/microsoft/FluidFramework/blob/main/build-tools/packages/build-cli/src/commands/generate/releaseNotes.ts)_
 
+## `flub generate source-entrypoints`
+
+Generates entrypoints for Fluid Framework API levels (/alpha, /beta, /public, /legacy) as found in package.json "exports"
+
+```
+USAGE
+  $ flub generate source-entrypoints [-v | --quiet] [--mainEntrypoint <value>]
+
+FLAGS
+  --mainEntrypoint=<value>  [default: ./src/index.ts] Main entrypoint file containing all untrimmed exports.
```

Comment:
Shouldn't this be read from the package?

Reply:
I guess this probably has to specified, since we are generating the entrypoint files based on the api levels of the apis in this file? I was thinking we would use the exports entries. I'm still wrapping my head around exactly what this is doing. :)

Reply:
yep. It includes paths containing all untrimmed exports. (which is usually `src/index.ts`)